openfecwebapp/templates/partials/individual-contributions-filter.html

```diff
@@ -0,0 +1,60 @@
+
+{% block heading %}
+Filter individual contributions
+{% endblock %}
+
+{% block message %}
+<div class="message message--info message--small">
+  <span class="t-block">Due to the large number of transactions, records begin in 2011.</span>
+</div>
+{% endblock %}
+
+{% block filters %}
+<div class="js-accordion accordion--neutral" data-content-prefix="filter" data-open-first="true">
+  <button type="button" class="js-accordion-trigger accordion__button">Recipient information</button>
+  <div class="accordion__content">
+    {{ typeahead.field('committee_id', 'Committee name or ID') }}
+  </div>
+  <button type="button" class="js-accordion-trigger accordion__button">Contributor information</button>
+  <div class="accordion__content">
+    <div class="filter">
+      <fieldset class="js-filter">
+        <legend class="label">Restrict contributions</legend>
+        <ul>
+          <li>
+            <div class="tooltip__container">
+              <input id="is-individual" name="is_individual" type="checkbox" value="true" class="tooltip-trigger" aria-describedby="unique-tooltip" checked>
+              <label for="is-individual" class="tooltip__trigger-text">Unique only</label>
+              <button class="tooltip__trigger" type="button"><span class="u-visually-hidden">Learn more</span></button>
+              <div id="unique-tooltip" role="tooltip" class="tooltip tooltip--under">
+                <p class="tooltip__content">Some transactions — such as earmarked contributions and partnership contributions — are reported more than once for clarity on the printed page but can lead to confusion when viewed as data online. Use the unique only filter to remove these extra transactions.</p>
+              </div>
+            </div>
+          </li>
+        </ul>
+      </fieldset>
+    </div>
+    {{ typeahead.field('contributor_name', 'Contributor name or ID', dataset='committees', allow_text=True) }}
+    {{ text.field('contributor_city', 'City') }}
+    {{ states.field('contributor_state') }}
+    {{ text.field('contributor_employer', 'Employer') }}
+    {{ text.field('contributor_occupation', 'Occupation') }}
+  </div>
+  <button type="button" class="js-accordion-trigger accordion__button">Transaction information</button>
+  <div class="accordion__content">
+    {{ text.field('min_amount', 'Minimum contribution', {'data-suffix': 'or more', 'data-inputmask': '"alias": "currency", "rightAlign": false, "clearMaskOnLostFocus": true' }) }}
+    {{ text.field('max_amount', 'Maximum contribution', {'data-suffix': 'or less', 'data-inputmask': '"alias": "currency", "rightAlign": false, "clearMaskOnLostFocus": true'}) }}
+    {{ date.field('date', 'Receipt date', dates ) }}
+    <div class="message message--info message--small">
+      <span class="t-block">Receipts are reported periodically, according to the filer's reporting schedule. Receipts are updated as they’re processed— that time can vary.</span>
+    </div>
+  </div>
+</div>
+{% endblock %}
```

Comment:
This is used by `receipts/individual-contributions`, which is the only new page this patch adds.